episodes/editing.md

```diff
@@ -142,6 +142,19 @@ These fields will be simple key-pair values of information used throughout the e
 carpentry
 : The code for the specific carpentry that the lesson belongs to (swc, dc, lc, cp)
 
+::::::::::: callout
+
+### Adding a custom logo
+
+The `carpentry` variable works with the `varnish` package to control the logo displayed on your lesson. You can display your organisation's logo by
```

Comment:
```suggestion
The "carpentry" variable works with the {varnish} package to control the logo displayed on your lesson. You can display your own logo by
```

I felt that 'organisation' was too specific, e.g. it could be a project logo instead.